test/depletion_test.jl

```diff
@@ -0,0 +1,25 @@
+using SolidStateDetectors
+using Test
+using Unitful
+
+T = Float32
+
+@testset "Test depletion estimation" begin
+    sim = Simulation{T}("BEGe_01.yaml")
+    calculate_electric_potential!(sim, refinement_limits=0.01)
+    id = SolidStateDetectors.determine_bias_voltage_contact_id(sim.detector)
+    calculate_weighting_potential!(sim, id, refinement_limits=0.01)
+    SolidStateDetectors._adapt_weighting_potential_to_electric_potential_grid!(sim, id)
```

Comment:
Is this actually needed in the tests or is it done automatically in `estimate_depletion_voltage` anyhow?